### DISCUSSION TITLE
(Pricing) - Flexible / Pay as you go pricing model

### DISCUSSION THREAD

**Discussion by 3stacks** [plausible/analytics#1856] [Anything Else]
G'day team.

I run short lived promotional websites which have very large fluctuations in public interest. In March I hit ~2 million page views but this month I'm on track for <100k page views. I have to increase and decrease my plan every other month or I may end up overpaying.

Cheers!

**Comment by metmarkosaric** [comment]
thanks for the feedback @3stacks! pay as you go would definitely help make payments more convenient for seasonal type of traffic. it's something we may consider in the future but for now there's the flexibility that you can go up or down in your plan at any point in time with a couple of clicks.

**Reply by gour** [reply]
Hello!

> pay as you go would definitely help make payments more convenient for seasonal type of traffic. it's something we may consider in the future... 

I'm running my own instance of Matomo and would consider switching to Plausible, but, currently, I pay 8€/mo for the hosting (500GB SSD, 2GB RAM...) to host few sites, so 9€/mo is simply too expensive for just analytics...so, pls. consider having some more pricing options ?

**Reply by metmarkosaric** [reply]
thanks for the feedback @gour! with our annual discount, the monthly cost goes down to €7.5. that's the best we can do at the moment. in addition to not having to worry about managing an analytics server, you also get advantage of our high availability, high capacity and a global CDN that makes sure the script loads fast no matter where your visitors are based. plus you support the development of open source software. thanks for understanding!

**Reply by 3stacks** [reply]
Hi @metmarkosaric just putting this one back in front of your eyes. We are on a custom enterprise plan so each time we have a seasonal burst we have to get in touch and get a custom plan created so it's not quite as seamless as it used to be for us. We end up overpaying when these seasonal projects end and then I have to bother Bogdan again to create a new plan for us. Cheers!
